retrieval_service/example-config-cloudsql.yml

```diff
@@ -1,7 +1,7 @@
 host: 0.0.0.0
 datastore:
   # Example for Cloud SQL
-  kind: "cloudsql-postgres"
+  kind: "cloudsql-dialect"
```

Comment:
Why is this dialect and not a specific engine type? They should still be different datastores right?

Reply:
Update this to be `cloudsql-engine`. The other configs is the same across cloudsql-postgres and cloudsql-mysql so I was thinking we can reduce the duplication of files by using the same ones. This is used during cloudbuild workflow to setup config.yml.